lib/ansible/module_utils/redfish_utils.py

```diff
@@ -275,7 +271,16 @@ def clear_logs(self):
                         return response
         return {'ret': True}
 
-    def get_storage_controller_inventory(self):
+    def aggregate(self, func):
+        ret = True
+        entries = []
+        for systems_uri in self.systems_uris:
+            inventory = func(systems_uri)
+            ret = inventory.pop('ret') and ret
+            entries.append(({'systems_uri': systems_uri}, inventory['entries']))
```

Comment:
In case no 'entries' were added, need to avoid a KeyError.
```suggestion
            if 'entries' in inventory:
                entries.append(({'systems_uri': systems_uri},
                                inventory['entries']))
```